Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -1,10 +1,18 @@
-# .NET Lambda Annotations Framework (Preview)
+# Amazon.Lambda.Annotations
```

Comment:
This is just the entire package README copied and pasted, don't think we need to re-review content here